src/core/core.ts

```diff
@@ -232,6 +232,8 @@ export class VConsole {
       this.compInstance.theme = theme;
     }
 
+    this.compInstance.lang = this.option.lang ? this.option.lang : "en"
```

Comment:
` this.compInstance.lang = this.option.lang ?? 'en';`

Reply:
yes